CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## Unreleased
 
+- ([GH-320](https://github.com/lingua-pupuli/puppet-vscode/issues/320)) Fix if/else snippet
```

Comment:
Might be useful to include the heading it will eventually go under after we update the subheadings for prior releases. Shouldn't hold up this PR.

Reply:
Good point. Done